************* Module aiocoap.cli.client
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:43:0: C0301: Line too long (257/240) (line-too-long)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:363:0: W0311: Bad indentation. Found 16 spaces, expected 12 (bad-indentation)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:36:4: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:86:4: C0415: Import outside toplevel (termcolor) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:91:8: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:106:8: C0415: Import outside toplevel (json) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:111:0: R0914: Too many local variables (16/15) (too-many-locals)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:122:8: C0415: Import outside toplevel (aiocoap.util.prettyprint.pretty_print) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:133:8: C0415: Import outside toplevel (aiocoap.util.prettyprint.lexer_for_mime) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:134:8: C0415: Import outside toplevel (pygments) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:149:8: C0415: Import outside toplevel (pygments.formatters.TerminalFormatter) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:150:8: C0415: Import outside toplevel (pygments.highlight) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:111:0: R0912: Too many branches (14/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:170:0: R0914: Too many local variables (20/15) (too-many-locals)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:193:12: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:193:12: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:204:4: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:205:8: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:208:8: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:212:12: E0237: Assigning to attribute 'accept' not defined in class slots (assigning-non-slot)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:215:16: E0237: Assigning to attribute 'accept' not defined in class slots (assigning-non-slot)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:217:16: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:217:16: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:220:8: E0237: Assigning to attribute 'observe' not defined in class slots (assigning-non-slot)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:225:12: E0237: Assigning to attribute 'content_format' not defined in class slots (assigning-non-slot)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:228:16: E0237: Assigning to attribute 'content_format' not defined in class slots (assigning-non-slot)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:230:16: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:230:16: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:236:16: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:238:16: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:241:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:242:16: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:246:20: C0415: Import outside toplevel (cbor2) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:247:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:248:20: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:249:16: C0415: Import outside toplevel (json) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:252:16: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:253:20: C0415: Import outside toplevel (ast) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:257:24: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:257:24: E0702: Raising NoneType while only classes or instances are allowed (raising-bad-type)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:263:8: E0237: Assigning to attribute 'block1' not defined in class slots (assigning-non-slot)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:283:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:286:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:290:8: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:170:0: R0912: Too many branches (42/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:170:0: R0915: Too many statements (119/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:319:4: C0103: Constant name "interactive_expecting_keyboard_interrupt" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:319:4: W0603: Using the global statement (global-statement)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:351:19: W0703: Catching too general exception Exception (broad-except)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:347:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:351:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:340:14: W0612: Unused variable 'pending' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:16:0: W0611: Unused import socket (unused-import)
/log/firmware/firmware/usr/lib/python3.8/site-packages/aiocoap/cli/client.py:23:4: W0611: Unused import readline (unused-import)

-----------------------------------
Your code has been rated at 6.01/10

d-argument)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:806:24: W0613: Unused argument 'data' (unused-argument)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:810:4: C0103: Method name "end_methodName" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:626:0: R0904: Too many public methods (21/20) (too-many-public-methods)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:840:8: R1720: Unnecessary "elif" after "raise" (no-else-raise)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:840:11: C0123: Using type() instead of isinstance() for a typecheck. (unidiomatic-typecheck)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:842:13: C0123: Using type() instead of isinstance() for a typecheck. (unidiomatic-typecheck)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:831:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:864:23: W0621: Redefining name 'server' from outer scope (line 1503) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:884:0: C0103: Constant name "FastMarshaller" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:884:17: C0103: Constant name "FastParser" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:884:30: C0103: Constant name "FastUnmarshaller" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:908:17: E1102: FastUnmarshaller is not callable (not-callable)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:909:17: E1102: FastParser is not callable (not-callable)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:913:21: E1102: FastParser is not callable (not-callable)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:964:8: C0103: Variable name "m" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:964:12: E1102: FastMarshaller is not callable (not-callable)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:966:8: C0103: Variable name "m" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1015:4: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1015:7: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1035:4: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1066:12: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1067:7: R1716: Simplify chained comparison between the operands (chained-comparison)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1087:8: C0103: Attribute name "io" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1082:23: W0621: Redefining name 'response' from outer scope (line 1515) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1157:12: C0103: Variable name "e" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1149:4: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1216:21: W0212: Access to a protected member _splituser of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1210:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1242:36: W0612: Unused variable 'x509' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1251:8: W0612: Unused variable 'host' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1288:4: R0201: Method could be a function (no-self-use)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1316:29: W0621: Redefining name 'response' from outer scope (line 1515) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1327:8: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1327:11: C0103: Variable name "u" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1168:16: W0201: Attribute 'verbose' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1417:8: W0622: Redefining built-in 'type' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1411:4: R0913: Too many arguments (8/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1450:8: W0621: Redefining name 'response' from outer scope (line 1515) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1479:8: R1705: Unnecessary "elif" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1507:4: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/xmlrpc/client.py:1517:4: C0103: Variable name "v" doesn't conform to snake_case naming style (invalid-name)

-----------------------------------
Your code has been rated at 8.31/10

